src/App.vue

```diff
@@ -1,54 +1,21 @@
 <template>
   <div>
-    <the-header></the-header>
+    <the-header />
     <main>
-      <carousel-list
-        v-if="carouselListData.length"
-        :carousel-list="carouselListData"
-      ></carousel-list>
-      <product-list :product-list="productList"></product-list>
+      <router-view></router-view>
```

Comment:
Nitpick, but doesn't eslint complain about a non-self closing component? 🤔 
```suggestion
      <router-view />
```

Reply:
eslint doesnt work, not in my power to setup it correctly :/